manifests-vaps/aws/0000_30_cluster-api_10_unsupported_aws_fields_validation.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: "unsupported-aws-fields"
+spec:
+  policyName: "unsupported-aws-fields"
+  validationActions: [Deny]
+  matchResources:
+    namespaceSelector:
+      matchLabels:
+        name: openshift-cluster-api
```

Comment:
This will need to change based on the findings at: https://github.com/openshift/cluster-capi-operator/pull/321/files#r2197472127